AWSIoT/Internal/AWSIoTMQTTClient.h

```diff
@@ -106,6 +108,11 @@
  */
 - (instancetype)initWithDelegate:(id<AWSIoTMQTTClientDelegate>)delegate;
 
+/**
+ Utility method to return username composed from usernameFields
```

Comment:
change "return username" to "user  metadata"

Reply:
Done